platform/golden/provisioners/01_update_packer_user/roles/packer-finalize/tasks/main.yml

```diff
@@ -1,39 +1,46 @@
 ---
 # Fix user setup for packer user
-
 #  lock password to avoid password logins
-
 - name: Lock packer user password to prevent password reset or login
   ansible.builtin.user:
     name: packer
     password: '*'
-
-#  update authorized keys from @sdake and @rstarmer
-
 - name: Set authorized keys for packer user - sdake
   ansible.posix.authorized_key:
     user: packer
     state: present
     key: "{{ lookup('url', 'https://github.com/sdake.keys', split_lines=False) }}"
-
-- name: Set authorized keys for packer user - sdake
+- name: Set authorized keys for packer user - rstarmer
   ansible.posix.authorized_key:
     user: packer
     state: present
     key: "{{ lookup('url', 'https://github.com/rstarmer.keys', split_lines=False) }}"
-
-# Update apt package cache and installed packages to latest versions
-
 - name: update apt packages
   ansible.builtin.apt:
     update_cache: true
     cache_valid_time: 3600
     upgrade: true
-
-# Fix grub config to support tty at boot
-
+- name: add iscsi packages
+  ansible.builtin.apt:
+    name: open-iscsi
+    state: present
+# Potential Fixes from https://github.com/oracle-quickstart/oci-byo-image/blob/master/scripts/debian_10.6/OCI.sh
```

Comment:
I have done several hours of research and placed that in [our wiki](https://github.com/artificialwisdomai/origin/wiki/Imaging). Can you take a look? You have studied the area in detail, maybe something is more apparent to you.